bin/linters.sh

```diff
@@ -35,16 +35,38 @@ function check_spelling() {
     echo 'spelling OK'
 }
 
+function has_latest_gometalinter() {
+    local local_binary
+    local last_release
+    local current_version
+
+    local_binary="${1}"
+    last_release=$(curl -L -s https://api.github.com/repos/alecthomas/gometalinter/releases/latest \
+	    | grep tag_name | sed "s/ *\"tag_name\": *\"\\(.*\\)\",*/\\1/" | sed "s/v//")
+    current_version="$(${local_binary} --version 2>/dev/null | cut -d ' ' -f 3)"
+
+    if [ "${last_release}" != "${current_version}" ]; then
+        return 1
+    fi
+
+    return 0
+}
+
 function install_gometalinter() {
+    gometalinter=$(command -v gometalinter 2> /dev/null || echo "${ISTIO_BIN}/gometalinter")
+    if has_latest_gometalinter "${gometalinter}"; then
+        echo "Skipping gometalinter installation, we already have the latest version"
+        return 0
+    fi
+
     echo 'Installing gometalinter ....'
-    go get -u gopkg.in/alecthomas/gometalinter.v2
-    if [[ -d "${ISTIO_BIN}/gometalinter.v2" ]];then
-        echo "download gopkg.in/alecthomas/gometalinter.v2 failed"
+    curl -s https://raw.githubusercontent.com/alecthomas/gometalinter/master/scripts/install.sh | bash -s -- -b "${ISTIO_BIN}"
```

Comment:
you looking for the latest verion, and then you install from master ? Should you be installing from the latest release version ?

Reply:
nice catch @ssebva - I missed that in the review.  Have removed /lgtm label.

Cheers
-steve